### DISCUSSION TITLE
How to set 30dBm ?

### DISCUSSION THREAD

**Discussion by f5bcb** [richonguzman/LoRa_APRS_iGate#197] [Q&A]
Hi,
My Lora APRS digi release is : 
esp32dev
-DESP32_DIY_1W_LoRa_Mesh_V1_2,-DHAS_SX1268,-DHAS_1W_LORA
Version date: 2024.08.14
Build date: 2024-08-14 18:25:41 UTC

My hamradio callsign is F5BCB
How to use the 30dBm Lora module ?
Thanks a lot
Jeff, F5BCB

**Comment by richonguzman** [comment]
E22 ebyte 400m30s has a LNA (low noise amplifier) so you set it up as 20db and it should output 30dB.

**Reply by f5bcb** [reply]
OK thanks a lot .... I'll check 
73', Jeff
